sat/cli/bootprep/documentation.py

```diff
@@ -117,7 +117,7 @@ def generate_docs_tarball(schema_path, output_dir):
         with tempfile.TemporaryDirectory() as tempdir:
             with cwd(tempdir), silence():
                 docs_output_path = os.path.join(tempdir, DEFAULT_HTML_INDEX)
-                generate_from_filename(schema_path, docs_output_path)
+                generate_from_filename(schema_path, docs_output_path, deprecated_from_description=True)
```

Comment:
Oh, I am not using this now. I'll get rid of this change, which was causing strange issues with logging for some reason too.